app/models/split.rb

```diff
@@ -152,6 +156,13 @@ def require_app_name_prefix?
     @require_app_name_prefix
   end
 
+  def decided_variant
+    return nil unless decided?
+    registry.each_with_object([]) { |(k, v), found|
+      found << k if v == 100
+    }.first
```

Comment:
Could this just become `registry.key(100)`?

Reply:
yes, didn't know or forgot that was a thing!